doc/nrf/ug_matter_creating_accessory.rst

```diff
@@ -304,6 +358,9 @@ For example, the implementation can look as follows:
            GetAppTask().PostEvent(AppEvent(*value ? AppEvent::SensorActivate : AppEvent::SensorDeactivate));
    }
 
+In this implementation, the ``if`` part filters out events other than those that belong to the On/Off cluster.
+Then, the callback posts the event for the sensor, namely ``SensorActivate`` if the current value of the attribute is not a zero.
```

Comment:
```suggestion
Then, the callback posts the event for the sensor, namely ``SensorActivate`` if the current value of the attribute is non-zero.
```
I'm not an expert, but "a zero" seems grammatically incorrect since there's only one zero in the world :D